capstone-frontend/src/components/UserCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Button, Col } from 'react-bootstrap';
+
+import '../styles/Groups.css';
+
+
+class UserCard extends Component {
+  removeMember = () => {
+    const jsonBody = {
+      "remove_memberIDs": this.props.user.id, 
+      "id": this.props.club.id
+    }
+    fetch(`/api/clubs`, {
+        method: "put", 
+        body: JSON.stringify(jsonBody)
+      })
+      .then(this.props.removeMember(this.props.user.id))
+  }
+
+  render() {
+    const removeMember = this.props.club 
+                             && this.props.club.ownerID === window.localStorage.getItem("userID")
+                             && this.props.club.ownerID !== this.props.user.id
```

Comment:
Nice checks for potential edge cases here! Just for clarification, are you ensuring that only the club owner can remove members, and additionally, that the club owner can't remove themselves from the club?

Reply:
Thanks! Yes, the idea is that only the club owner can remove members. Additionally, the club owner should not be able to be removed from the club (even though the only person who can do this is themselves).